assignments/closure.js

```diff
@@ -1,19 +1,45 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+let myName = "Josh";
+
+function sayHello() {
+  let example = 'jim';
+  return `Hello ${myName}`;
+}
+
+console.log(sayHello());
+console.log(example);
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
```

Comment:
Did you grab this from the solution or from a fellow student, or did you write it yourself? Either is fine, it's just good for me to know.